Documentation/user-guides/cluster-monitoring.md

```diff
@@ -480,6 +480,49 @@ spec:
 
 Read more in the [alerting guide](alerting.md) on how to configure the Alertmanager as it will not spin up unless it has a valid configuration mounted through a `Secret`. Note that the `Secret` has to be in the same namespace as the `Alertmanager` resource as well as have the name `alertmanager-<name-of-alertmanager-object` and the key of the configuration is `alertmanager.yaml`.
 
+## Basic auth
```

Comment:
I think this should be a separate user-guide documentation. Called something like "Basic Auth for targets", and the way the `embedmd` things work is that the file content is synced into the documentation with the `make generate` make target. So just make sure those files exist and then run that make target, then the content will be synced (we test that the documentation is consistent in one of the jenkins jobs).